.github/scripts/Rename_Photos

```diff
@@ -0,0 +1,26 @@
+#This script renames photos from BHVP_PH_CetaitParis_DP_1538_01_015 and BHVP_PH_CetaitParis_DP_1538_01_016.jpg to 16_slide.jpg and 16_photo.jpg
+
+import math
+import os
+import csv
+
+folder = r"/Users/ismailassafi/Documents/GitHub/paris_1970/assets/images/photos/1538/"
```

Comment:
I think this is the old file?  I remember seeing another file with a function with arguments to call instead of a hard-coded folder path

Reply:
Yes, sorry. Let me change this right now.